packages/transformers/sass/src/SassTransformer.js

```diff
@@ -0,0 +1,96 @@
+// @flow
+
+import {Transformer} from '@parcel/plugin';
+import localRequire, {localResolve} from '@parcel/local-require';
```

Comment:
Unfortunately this will break with the latest changes introduced by #3320.

Starting with #3320, we pass `localRequire` as an option bound to the current workerfarm instance (we need it since we have to centralize auto-installs in the main process). We'll need to do something similar with `localResolve`.

Reply:
The node-sass detection doesn't need the ability to auto install, or to require even, only resolution, maybe we can use `resolve` from `@parcel/utils`?